rollup/app/lwc/rollupForceRecalculation/rollupForceRecalculation.html

```diff
@@ -1,77 +1,93 @@
-          onchange={handleChange}
-          label="Where Clause SOQL Filters (Optional)"
-          name="potentialWhereClause"
-        ></lightning-textarea>
+        <template if:true={isCMDTRecalc}>
+          <lightning-combobox
+            name="Select Calc Item"
+            label="Select Calc Item"
+            value={selectedMetadata}
+            options={rollupMetadataOptions}
+            onchange={handleComboChange}
+          ></lightning-combobox>
+        </template>
+        <template if:false={isCMDTRecalc}>
+          <lightning-input
+            class="slds-col slds-form-element slds-form-element_horizontal"
+            type="text"
+            label="Calc Item SObject API Name"
+            name="calcItemSObjectName"
+            onchange={handleChange}
+            required
+          >
+          </lightning-input>
+          <lightning-input
+            class="slds-col slds-form-element slds-form-element_horizontal"
+            type="text"
+            label="API Name of the rollup field on calc item"
+            name="opFieldOnCalcItem"
+            onchange={handleChange}
+            required
+          >
+          </lightning-input>
+          <lightning-input
+            class="slds-col slds-form-element slds-form-element_horizontal"
+            type="text"
+            label="API Name of the lookup field on calc item"
+            name="lookupFieldOnCalcItem"
+            onchange={handleChange}
+            required
+          >
+          </lightning-input>
+          <lightning-input
+            class="slds-col slds-form-element slds-form-element_horizontal"
+            type="text"
+            label="API Name of the lookup field on the lookup object"
+            name="lookupFieldOnLookupObject"
+            onchange={handleChange}
+            required
+          >
+          </lightning-input>
+          <lightning-input
+            class="slds-col slds-form-element slds-form-element_horizontal"
+            type="text"
+            label="API Name of the rollup field on the lookup object"
+            name="rollupFieldOnLookupObject"
+            onchange={handleChange}
+            required
+          >
+          </lightning-input>
+          <lightning-input
+            class="slds-col slds-form-element slds-form-element_horizontal"
+            type="text"
+            label="Lookup SObject API Name"
+            name="lookupSObjectName"
+            onchange={handleChange}
+            required
+          >
+          </lightning-input>
+          <lightning-input
+            class="slds-col slds-form-element slds-form-element_horizontal"
+            type="text"
+            label="Rollup Operation Name (SUM/MIN/MAX/COUNT/COUNT_DISTINCT/CONCAT/CONCAT_DISTINCT/AVERAGE/FIRST/LAST)"
+            name="operationName"
+            onchange={handleChange}
+            required
+          >
+          </lightning-input>
+          <lightning-textarea
```

Comment:
Is it possible to show this `WHERE` filter for the CMDT view too? You may want to run all CMDT rollups for just a subset of records, but the current UI doesn't seem to support it.

![image](https://user-images.githubusercontent.com/1267157/112763191-49cf2a00-8fb8-11eb-9dd7-16a8e1eda3a3.png)

Reply:
(Jonathan and I chatted offline about this one): the where clause comes from each of the `Rollup__mdt` records, so we don't need to retain the text area field from the first part of the LWC